pkg/componentreadiness/jiraautomator/jiraautomator.go

```diff
@@ -0,0 +1,597 @@
+		values.Add("ignoreDisruption", "1")
+	} else {
+		values.Add("ignoreDisruption", "0")
+	}
+	if reportOpts.AdvancedOption.IgnoreMissing {
+		values.Add("ignoreMissing", "1")
+	} else {
+		values.Add("ignoreMissing", "0")
+	}
+	values.Add("minFail", fmt.Sprintf("%d", reportOpts.AdvancedOption.MinimumFailure))
+	values.Add("pity", fmt.Sprintf("%d", reportOpts.AdvancedOption.PityFactor))
+
+	absURL += values.Encode()
+	viewURL := j.sippyURL + "/sippy-ng/component_readiness/main?view=" + view.Name
+	return absURL, viewURL, nil
+}
+
+// createNewJiraIssueForRegressions creates new issue for components by
+// a. setting the ticket's Affects Version/s= sample version.
+// b  adding the Regression label defined by LabelJiraAutomator.
+// c. setting a description with links to CR
+// d. for pre-release, setting "Release Blocker" label to Approved
+func (j JiraAutomator) createNewJiraIssueForRegressions(view crtype.View, component string, tests []crtype.ReportTestSummary, linkedIssue *jira.Issue) (*jira.Issue, error) {
+	if len(tests) > 0 {
+		description := `Component Readiness has found a potential regression in the following tests:`
+		for i, test := range tests {
+			// Only show stats for the worst regression
+			if i == 0 {
+				description += fmt.Sprintf("\n h4. Most Regressed Test:\n{code}%s{code}\n", test.TestName)
+				description += getProbabilityString(test.ReportStatus, *test.FisherExact)
+				description += getStatsString("Sample (being evaluated)", test.SampleStats, view.SampleRelease.RelativeStart, view.SampleRelease.RelativeEnd)
+				description += getStatsString("Base (historical)", *test.BaseStats, view.BaseRelease.RelativeStart, view.BaseRelease.RelativeEnd)
+				if len(tests) > 1 {
+					description += fmt.Sprintf("\n h4. Other Regressed Tests:\n")
+					description += fmt.Sprintf("\nThe following tests are also regressed in the same component readiness report. They might not be related to the most regressed test above. We only create one issue per component and therefore group them here. Feel free to create new issues if they are unrelated.\n")
+				}
+			} else {
+				description += fmt.Sprintf("{code}%s{code}\n", test.TestName)
+				description += getProbabilityString(test.ReportStatus, *test.FisherExact)
+			}
+		}
+		if linkedIssue != nil {
+			description += fmt.Sprintf("\n h4. Potentially Related Issues:\n")
+			description += fmt.Sprintf("\n* This regression might be related to [%s|%s]. Feel free to link it if found related.\n", linkedIssue.Key, linkedIssue.Self)
+		}
+
+		absUrl, viewUrl, err := j.getComponentReadinessURLsForView(view)
+		if err != nil {
+			return nil, err
+		}
+		description += fmt.Sprintf("\n h4. Useful Component Readiness Links:\n")
+		description += fmt.Sprintf("\nWe are proving the following two links for your convenience:\n")
+		description += fmt.Sprintf("\n- Click [here|%s] to access the component readiness page generated at the time this issue was created.\n", absUrl)
```

Comment:
suggest: "to access a snapshot of the component readiness page at the time this issue was generated"